tests/infer/mcmc/test_valid_models.py

```diff
@@ -463,6 +468,65 @@ def model():
             pyro.sample("obs",
                         dist.ExtendedBinomial(i, response_rate),
                         obs=data)
+        logger.debug(" ".join(map("{:0.3g}".format, i)))
+
+    mcmc_kernel = kernel(model, max_plate_nesting=1, **kwargs)
+    assert_ok(mcmc_kernel)
+
+
+@pytest.mark.parametrize("kernel, kwargs", [
+    (HMC, {"adapt_step_size": True, "num_steps": 3}),
```

Comment:
@fritzo you can set `trajectory_length=1` to reduce the number of `num_steps`. In HMC, `num_steps` is a dynamic value:
```
num_steps = trajectory_length / step_size
```